addons/sys_core/fnc_getGear.sqf

```diff
@@ -24,7 +24,7 @@ if (isNull _unit) exitWith {[]};
 private _gear = (getItemCargo (uniformContainer _unit)) select 0;
 _gear append ((getItemCargo (vestContainer _unit)) select 0);
 _gear append ((getItemCargo (backpackContainer _unit)) select 0);
-_gear pushBack ((assignedItems _unit) select 3);
+_gear append (assignedItems _unit);
```

Comment:
```suggestion
_gear pushBack ((assignedItems _unit) param [3, ""]);
```

Reply:
Hmmm no, I would  not risk  it. From the description: https://community.bistudio.com/wiki/assignedItems
>But if something is missing it get's omitted so you can not be sure that some element is at a constant index. 

So it may be our "item radios" are on position 1...

Reply:
I have never seen that happen before. :thinking: